UKHomeOffice/cerberus-service
# COP-10438 - Task details booking

## Description
This PR adds the booking section within task details for a task.

## To Test
- Pull and run against dev
- Within `../routes/airpax/TaskDetails/TaskDetailsPage.jsx` on `line 30`, paste in the contents of the fixture file `../routes/airpax/TaskDetails/__fixtures__/taskData_AirPax_AssigneeCurrentUser.fixture.json`
- in the browser, go to `localhost:8080/airpax/tasks/DEV-DEV-20220419-001`
- Confirm that the booking section within the second column, is rendered, similar to the screen shot below.
 
![image](https://user-images.githubusercontent.com/19333750/165751715-156a7a18-c0e8-436d-a779-ecd2607e7b97.png)


## Developer Checklist

\* Required
